.travis.yml

```diff
@@ -12,7 +12,8 @@ install:
   - pip install coveralls
 script: 
   - tox
-after_success: coveralls
+after_success:
+  - coveralls --merge=coverage/coverage-js.json
```

Comment:
I looked at the coveralls for the latest job on this branch and it contains both python & JS!: https://coveralls.io/jobs/6776876